### DISCUSSION TITLE
Way to highlight all search results in SumatraPDF

### DISCUSSION THREAD

**Discussion by Bread-King** [sumatrapdfreader/sumatrapdf#4189] [Q&A]
Can I see all the search results and highlight the current one in a PDF? just like the figure paste below
This feature will be valuable for reading some literature.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/65778786/0d0e6f9b-88da-4c99-9d42-f86af17ba7a5)

**Comment by kjk** [comment]
There's an issue tracking this: https://github.com/sumatrapdfreader/sumatrapdf/issues/154
Which is not a promise that it'll happen any time soon.

**Comment by Bread-King** [comment]
Thanks!
I thought that this kind of functionality could be achieved through other customizations.

**Reply by GitHubRulesOK** [reply]
The way text is stored as binary numbers in a PDF means there is no guarantee each letter pair such as you show  `to` will always be the same 2 sequential digits thus scanning a page or a file for 2 characters is itself a hard task in a PDF. take this example 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7636ca6b-c75a-4299-b601-9264adc4e962) code is here 
https://stackoverflow.com/a/78330304/10802527 test file is here 
[abracadabra.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/15005055/abracadabra.pdf)
